porting/debugging/configuration/networking.rst

```diff
@@ -0,0 +1,213 @@
+
+Key service configurations::
+
+    # RIL (Radio Interface Layer)
+    systemctl status ofono
+    systemctl status urfkill
+    
+    # WiFi services
+    systemctl status wpa_supplicant
+    systemctl status network-manager
+
+Understanding Network Architecture
+----------------------------------
+
+Network Subsystems
+^^^^^^^^^^^^^^^^^^
+Ubuntu Touch networking consists of three main subsystems:
+
+1. **WiFi Stack**
+   
+   * Kernel WiFi drivers
+   * wpa_supplicant (authentication)
+   * NetworkManager (connection management)
+   * Firmware blobs
+
+2. **Mobile Data**
+   
+   * RIL (Radio Interface Layer)
+   * oFono (telephony framework)
+   * RNDIS (USB networking)
+   * Modem firmware
+
+3. **Network Management**
+   
+   * NetworkManager
+   * DNS resolution
+   * Network routing 
+   * Interface configuration
+
+Configuration Steps
+-------------------
+
+1. WiFi Configuration
+^^^^^^^^^^^^^^^^^^^^^
+Set up WiFi support in your kernel configuration::
+
+    # Required kernel configs
+    CONFIG_WIRELESS=y              # Wireless subsystem support
+    CONFIG_CFG80211=y             # cfg80211 wireless interface
+    CONFIG_MAC80211=y             # IEEE 802.11 support
+    CONFIG_WLAN=y                 # Wireless LAN drivers
+
+Common vendor-specific configs::
+
+    # Qualcomm WiFi
+    CONFIG_QCA_CLD_WLAN=y
+    
+    # MediaTek WiFi
+    CONFIG_MTK_COMBO_WIFI=y
+    
+    # Broadcom WiFi
+    CONFIG_BCM4339=y
+
+Firmware setup::
+
+    # Common firmware locations
+    /vendor/firmware/              # Vendor firmware directory
+    └── wifi/
+        ├── firmware.bin          # Main firmware
+        ├── nvram.txt            # Radio parameters
+        └── board.bin            # Board-specific data
+
+2. Mobile Data Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Configure mobile data support:
+
+Kernel configuration::
+
+    # USB networking
+    CONFIG_USB_NET_RNDIS_HOST=y    # RNDIS host support
+    CONFIG_USB_CONFIGFS_RNDIS=y    # ConfigFS RNDIS gadget
+    
+    # Modem support
+    CONFIG_USB_SERIAL=y            # USB Serial support
+    CONFIG_USB_SERIAL_WWAN=y       # USB Serial WWAN driver
+    CONFIG_USB_SERIAL_OPTION=y     # USB Serial Option driver
+
+RIL configuration::
+
+    # /etc/ofono/ril_subscription.conf
+    [Settings]
+    Subscribe=false
+    
+    [ril_0]
+    transport=binder
+    socket=/dev/socket/rild
+    name=slot1
+
+    [ril_1]                        # For dual-SIM devices
+    transport=binder
+    socket=/dev/socket/rild2
+    name=slot2
+
+3. Network Management Setup
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Configure NetworkManager for interface handling::
+
+    # /etc/NetworkManager/NetworkManager.conf
+    [main]
+    plugins=ifupdown,keyfile
+    
+    [ifupdown]
+    managed=true
+    
+    [device]
+    wifi.scan-rand-mac-address=no
+
+DNS configuration::
+
+    # /etc/systemd/resolved.conf
+    [Resolve]
+    DNS=1.1.1.1 8.8.8.8
+    FallbackDNS=9.9.9.9
+    DNSSEC=allow-downgrade
+    
```

Comment:
Why porters should touch these configs?

Reply:
I can't answer that. This is in essence why the reworked material needs scrutiny by someone with expert knowledge of what is going on.

On a general note: Some of the information was purposefully included with the goal of improving understanding and not necessarily to invite people to change the settings. Again, remove, or keep and specify something like "not to be changed"?